avocado/utils/memory.py

```diff
@@ -41,17 +41,17 @@ def _check_memory_state(block):
     Check the given memory block is online or offline
 
     :param block: memory block id.
-    :type string: like 198
+    :type string: like memory198
```

Comment:
Please don't do incompatible changes like this as it breaks existing tests. I agree such usage is perhaps more natural and easier to use, so how about to accept both?

```python
if not block.startswith("memory"):
    block = "memory%s" % block
```